Doc/library/test.rst

```diff
@@ -1691,6 +1691,18 @@ The :mod:`test.support.warnings_helper` module provides support for warnings tes
 .. versionadded:: 3.10
 
 
+.. function:: ignore_warnings(*, category)
+
+   A decorator to suppress warnings. *category* is a subclass of :exc:`Warning`
+   class. It is approximately equivalent to calling :func:`warnings.catch_warnings()`
+   with :meth:`warnings.simplefilter` set to ``ignore``. If you want to suppress some
+   specific known warnings in a test case, use code like the following::
+
+      @warning_helper.ignore_warnings(category=DeprecationWarning)
+      def test_suppress_warning():
+          # do something
```

Comment:
```suggestion
          # do something

   .. versionadded:: 3.8          
```

Add versionadded per @erlend-aasland 's comment (sorry, though it was added in 3.10 like the rest, but in fact the notation above was due to them being moved to a new submodule instead).